### PR TITLE
Add deno-solid template

### PR DESCRIPTION
**Pull request by JLarky**
image

I tested `dev/build/preview/serve` with deno 1.26.0 because of this [bug](https://twitter.com/parasocialML/status/1581884543979515906) with 1.26.1

### CODE REVIEW

File: template-deno-solid/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          count is {count}
+        </button>
+        <p>
+          Edit <code>src/App.jsx</code> and save to test HMR
+        </p>
+      </div>
+      <p className="read-the-docs">
+        Click on the Vite and Solid logos to learn more
+      </p>
+    </div>
```
Comment (bluwy) [inline template-deno-solid/src/App.jsx:31]: Should we use `class` instead of `className` here?

**Comment by bluwy** [comment]
Thanks!

**Comment by bluwy** [comment]
Released in 0.1.8. I also added solid-ssr templates.

**Comment by cnava9389** [comment]
I get an "error when starting a dev server: Cannot find package.json from **/**/<my-project>". on a fresh install using deno 1.26.0 and 1.27.2. any ideas?

**Pull request merged**